Source/DataSources/StaticGeometryColorBatch.js

```diff
@@ -208,22 +208,22 @@ define([
 
                 if (!updater.fillMaterialProperty.isConstant || waitingOnCreate) {
                     var colorProperty = updater.fillMaterialProperty.color;
-                    colorProperty.getValue(time, colorScratch);
-                    if (!Color.equals(attributes._lastColor, colorScratch)) {
-                        attributes._lastColor = Color.clone(colorScratch, attributes._lastColor);
-                        attributes.color = ColorGeometryInstanceAttribute.toValue(colorScratch, attributes.color);
+                    var resultColor = colorProperty.getValue(time, colorScratch);
```

Comment:
This is a perfect example for why I think we should always prefer 

```js 
var result = someFunction(resultParameter)
// do more things with result
``` 
instead of 
```js
someFunction(resultParameter);
//do more things with resultParameter
```

I think it makes the code easier to follow for someone who is not used to how we use result parameters, and it avoids mistakes like this where the result parameter might not actually be used.

Reply:
I agree with you.  I think this is in our coding guide, can you check and open a PR if not?  Thanks.